package.json

```
@@ -10,13 +10,15 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "CI=true react-scripts test"
+    "test": "CI=true react-scripts test",
+    "test:windows": "cross-env CI=true react-scripts test"
```

The whole point of `cross-env` is to avoid having separate commands per platform
```suggestion
    "test": "cross-env CI=true react-scripts test"
```

---

👍